### DISCUSSION TITLE
Where to find vocabulary?

### DISCUSSION THREAD

**Discussion by nathanbowang** [apache/jena#1429] [Q&A]
### Version

4.5.0

### Question

Where is the vocabulary of the below ontology? 
```ttl
PREFIX fuseki:  <http://jena.apache.org/fuseki#>
PREFIX tdb2:    <http://jena.apache.org/2016/tdb#>
PREFIX ja:      <http://jena.hpl.hp.com/2005/11/Assembler#>
```

URLs are not accessible. 

There is some source code relative, but they are all missing elements in the configuration examples. 

- src/main/java/org/apache/jena/fuseki/server/FusekiVocab.java missing:
`fuseki:dataset`
- src/main/java/org/apache/jena/assembler/JA.java missing 
`ja:RDFDataset`
- src/main/java/org/apache/jena/tdb2/TDB2.java missing:
`tdb2:GraphTDB2`
`tdb2:DatasetTDB2`
`tdb2:dataset`
`tdb2:location`

**Comment by afs** [comment]
The project does not have access to `http://jena.hpl.hp.com/` any more (pre-Apache situation).

`VocabTDB2` has the TDB vocabulary declarations used by the assembler.

`DatasetAssemblerVocab` has the general dataset vocabulary. It is squatting on JA (but JA is in jena-core).

`fuseki:dataset` is an omission - although the service description itself is not handled by assemblers due to its complexity and also it is not a general element that can be combined with other things.
